collections/app/lib/Config.scala

```diff
@@ -0,0 +1,30 @@
+package lib
+
+import com.amazonaws.regions.{Regions, Region}
+import com.gu.mediaservice.lib.config.{Properties, CommonPlayAppConfig, CommonPlayAppProperties}
+import com.amazonaws.auth.{BasicAWSCredentials, AWSCredentials}
+
+
+object Config extends CommonPlayAppProperties with CommonPlayAppConfig {
+
+  val appName = "collections"
+
+  val properties = Properties.fromPath("/etc/gu/collections.properties")
+
+  val awsCredentials: AWSCredentials =
+    new BasicAWSCredentials(properties("aws.id"), properties("aws.secret"))
+
+  val dynamoRegion: Region = Region.getRegion(Regions.EU_WEST_1)
```

Comment:
I would not spend too much time worrying about that. That's an easy PR if someone is interested.